app/models/manageiq/providers/ibm_cloud/cloud_object_storage/storage_manager.rb

```diff
@@ -0,0 +1,37 @@
+class ManageIQ::Providers::IbmCloud::CloudObjectStorage::StorageManager < ManageIQ::Providers::StorageManager
+  require_nested :MetricsCapture
+  require_nested :MetricsCollectorWorker
+  require_nested :Refresher
+  require_nested :RefreshWorker
+
+  supports :ems_storage_new
+  include ManageIQ::Providers::StorageManager::ObjectMixin
+
+  def self.ems_type
+    @ems_type ||= "cloud_object_storage".freeze
+  end
+
+  def self.description
+    @description ||= "Cloud Object Storage".freeze
```

Comment:
Same, this should probably be `IBM Cloud Object Storage`

Reply:
same, I'll rename now